system/helpers.php

```diff
@@ -53,9 +53,10 @@ function dd() {
  * @return string
  */
 function noise($size = 32) {
-	$pool = 'abcefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890';
-
-	return substr(str_shuffle(str_repeat($pool, 3)), 0, $size);
+	return Security::randomString(
+		$size,
+		'abcefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890'
+	);
```

Comment:
The same with these lines. Readability would be greatly improved. Generally, I only break things up like this when I have a big array, usually of >7 items.